doc/guides/node-postmortem-support.md

```diff
@@ -15,8 +15,8 @@ Node.js always includes these constants in the final build.
 
 ### Node.js Debug Symbols
 
-Node prefixes all postmortem constants with `nodedbg_`, and they complement V8
-constants by providing ways to inspect Node-specific structures, like
+Node.js prefixes all postmortem constants with `nodedbg_`, and they complement V8
```

Comment:
This line has 81 characters. Why is linter green?

Reply:
Yes, it checks guides. Surprisingly, we don't lint for lines longer than 80 chars in markdown files.